### ISSUE TITLE
latest version: loss become nas when train retinanet by using default setting

### ISSUE THREAD

**Issue by charlesyann** [open-mmlab/mmdetection#2724]
when i train retinanet with default setting:
python tools/train.py configs/retinanet/retinanet_r50_fpn_1x_coco.py

loss became nan very quickly with latest master version:

2020-05-14 21:45:46,912 - mmdet - INFO - Epoch [1][300/58633]   lr: 0.00598, eta: 1 day, 13:22:01, time: 0.190, data_time: 0.007, me
mory: 3326, loss_cls: 1.1986, loss_bbox: 0.6736, loss: 1.8722
2020-05-14 21:45:56,289 - mmdet - INFO - Epoch [1][350/58633]   lr: 0.00698, eta: 1 day, 13:15:40, time: 0.188, data_time: 0.007, me
mory: 3326, loss_cls: 1.2054, loss_bbox: 0.6846, loss: 1.8900
2020-05-14 21:46:05,889 - mmdet - INFO - Epoch [1][400/58633]   lr: 0.00798, eta: 1 day, 13:17:22, time: 0.192, data_time: 0.007, me
mory: 3326, loss_cls: 1.2191, loss_bbox: 0.6791, loss: 1.8982
2020-05-14 21:46:15,403 - mmdet - INFO - Epoch [1][450/58633]   lr: 0.00898, eta: 1 day, 13:16:22, time: 0.190, data_time: 0.007, me
mory: 3326, loss_cls: 1.1870, loss_bbox: 0.6732, loss: 1.8602
2020-05-14 21:46:24,804 - mmdet - INFO - Epoch [1][500/58633]   lr: 0.00998, eta: 1 day, 13:12:54, time: 0.188, data_time: 0.007, me
mory: 3326, loss_cls: 1.2092, loss_bbox: 0.6594, loss: 1.8685
2020-05-14 21:46:34,364 - mmdet - INFO - Epoch [1][550/58633]   lr: 0.01000, eta: 1 day, 13:13:18, time: 0.191, data_time: 0.007, me
mory: 3326, loss_cls: 1.1759, loss_bbox: 0.6573, loss: 1.8333
2020-05-14 21:46:43,794 - mmdet - INFO - Epoch [1][600/58633]   lr: 0.01000, eta: 1 day, 13:11:16, time: 0.189, data_time: 0.007, me
mory: 3326, loss_cls: 1.2104, loss_bbox: 0.6845, loss: 1.8949
2020-05-14 21:46:53,110 - mmdet - INFO - Epoch [1][650/58633]   lr: 0.01000, eta: 1 day, 13:07:22, time: 0.186, data_time: 0.007, me
mory: 3326, loss_cls: nan, loss_bbox: 0.7355, loss: nan
2020-05-14 21:47:02,872 - mmdet - INFO - Epoch [1][700/58633]   lr: 0.01000, eta: 1 day, 13:11:31, time: 0.195, data_time: 0.010, me
mory: 3326, loss_cls: nan, loss_bbox: 0.6842, loss: nan

**Comment by ZwwWayne** [comment]
The batch size is very small, you need to modify the learning rate.
The LR is set assuming the 8-GPU training (batch size = 8x2=16), you need to modify it according to your batch size.

**Comment by FreyWang** [comment]
I met the same problem when train SSD, modify lr or max_norm did not work, It seems some bug exsit in V2.0

**Comment by ZwwWayne** [comment]
Hi @FreyWang ,
Please create a new  issue following the `error template` and describe your `same problem` or `did not work` in detail. This issue is mainly about large LR for small batch size.

**Comment by FreyWang** [comment]
> Hi @FreyWang ,
> Please create a new issue following the `error template` and describe your `same problem` or `did not work` in detail. This issue is mainly about large LR for small batch size.

Hi, thank your for your work ,I have found the loss_nan bug when using dirty data and I will commit a PR today
